api/v1alpha1/nonadminbackup_types.go

```diff
@@ -39,19 +60,28 @@ type NonAdminBackupSpec struct {
 
 // NonAdminBackupStatus defines the observed state of NonAdminBackup
 type NonAdminBackupStatus struct {
-	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	// OadpVeleroBackup references the VeleroBackup object. Format: <name>.<namespace>
+	// +optional
+	OadpVeleroBackup string `json:"oadpVeleroBackup,omitempty"`
```

Comment:
just `VeleroBackup`? or `VeleroBackupRef`?

Reply:
I wanted to have clear indication that it's VeleroBackup, but the one created by OADP controller, so not any VeleroBackup, but it has to be a VeleroBackup by NAC. Maybe VeleroBackup is fine? Just a bit confusing. Ref also seems a bit too techy